gui/control_main.py

```diff
@@ -2693,13 +2702,24 @@ def moveOmegaCB(self):
             {"relative": False}
         )
 
-    def moveEnergyCB(self):
+    def moveEnergy10eVCB(self):
         energyRequest = float(str(self.energy_ledit.text()))
-        if abs(energyRequest - self.energy_pv.get()) > 10.0:
-            self.popupServerMessage("Energy change must be less than 10 ev")
-            return
+        if self.controlEnabled():
+            if abs(energyRequest - self.energy_pv.get()) > 10.0:
+                self.popupServerMessage("Energy change must be less than 10 ev")
+                return
+            else:
+                self.send_to_server("mvaDescriptor", ["energy", float(self.energy_ledit.text())])
+                comm_s = 'mvaDescriptor("energy",' + str(self.energy_ledit.text()) + ")"
```

Comment:
move this comm_s definition up and send_to_server

Reply:
comm_s is for logging purposes only. send_to_server does not take the same format